source/isaaclab/isaaclab/envs/manager_based_env.py

```diff
@@ -132,6 +136,16 @@ def __init__(self, cfg: ManagerBasedEnvCfg):
         else:
             self.viewport_camera_controller = None
 
+        # create event manager
+        # note: this is moved here to allow USD-related randomization events that need to be
+        #   applied before the simulation starts.
```

Comment:
```suggestion
        # note: this is needed here (rather than after simulation play) to allow USD-related randomization events
        #   that must happen before the simulation starts. Example: randomizing mesh scale
```